scripts/set_151_officer.py

```diff
@@ -0,0 +1,130 @@
+"""
+==============
+Pre-requisites
+=============
+Set the below environmnetal variable
+export FLASK_ENV=development
+export ACCOUNT_STORE_API_HOST=http://localhost:3003
+
+==============
+To Run
+=============
+# Create or update an account with the role "OFFICER_151"
+>> python -m scripts.set_151_officer <email_address1> <email_address2> ...
```

Comment:
I would be vaguely in favour of making this a Flask CLI command instead and making that our consistent way of doing management commands for our services (see data-store with the `flask db-data` and `flask admin` commands.

Maybe this could be a `flask admin set-151-officers` command? Then we have a bit of a framework for how to build the commands, do command line args/flags, docstrings etc.

What do you think? It should be _mostly_ a lift and shift and then adding some Click decorators, but there'd probably be some reworking too.